spyder/plugins/explorer/tests/test_explorer.py

```diff
@@ -97,9 +102,64 @@ def test_project_explorer(project_explorer):
     assert project_explorer
 
 
-def test_delete_files_folders(explorer_with_files, mocker):
+@pytest.mark.parametrize('path_method', ['absolute', 'relative'])
+def test_copy_path(explorer_with_files, path_method):
+    """Test copy absolute and relative paths."""
+    project, __, file_paths, __, cb = explorer_with_files
+    explorer_directory = project.explorer.treewidget.fsmodel.rootPath()
+    explorer_directory = ('os.sep'.join(explorer_directory.strip('os.sep').
+                                        split('os.sep')[1:]))
+    project.explorer.treewidget.copy_path(fnames=file_paths,
+                                          method=path_method)
+    cb_output = cb.text(mode=cb.Clipboard)
+    path_list = [path.strip(',"') for path in cb_output.splitlines()]
+    assert len(path_list) == len(file_paths)
+    for path, expected_path in zip(path_list, file_paths):
+        assert osp.exists(path)
+        if path_method == 'relative':
+            expected_path = osp.relpath(expected_path, explorer_directory)
+        assert osp.normpath(path) == osp.normpath(expected_path)
+        if osp.isfile(path) and path_method == 'absolute':
```

Comment:
You can remove this line (``if osp.isfile(path) and path_method == 'absolute':``) and everything below it, since checking the contents isn't needed for this test of copying the paths.